packages/devtools/src/constants.ts

```diff
@@ -77,7 +77,6 @@ export const DEFAULTS: Options.Definition<Options.WebDriver> = {
      */
     logLevel: {
         type: 'string',
-        default: 'info',
```

Comment:
Why removing the default values?

Reply:
I mentioned above, in "proposed changes"

> 
Currently if you set WDIO_LOG_LEVEL env variable, you will still see `info` level logs from `devtools` and `webdriver` because `devtools.newSession` and `webdriver.newSession` defaulting `logLevel` to `info` ([devtools](https://github.com/webdriverio/webdriverio/blob/main/packages/devtools/src/index.ts#L40) and [webdriver](https://github.com/webdriverio/webdriverio/blob/main/packages/webdriver/src/index.ts#L23)). Though its not needed because  `setLogLevelsConfig` already sets right logLevels on `wdio.remote` [here](https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/src/index.ts#L44), and `getLogger` [already has this fallback with default value](https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-logger/src/node.ts#L120).

Reply:
Let's keep this anyway as it better describes the default behavior. The default log level for `@wdio/logger` can maybe change in the future and we would like to keep the current default in `devtools` and `webdriver`.

Reply:
Whats about

```ts
    options.logLevel = process.env.WDIO_LOG_LEVEL ?? options.logLevel
    const params = validateConfig(DEFAULTS, options)
```
[in devtools/src/index.ts](https://github.com/webdriverio/webdriverio/blob/main/packages/devtools/src/index.ts#L40)

or

```ts
    const params = validateConfig(DEFAULTS, options)
    params.logLevel = process.env.WDIO_LOG_LEVEL?? options.logLevel ?? params.logLevel
```

to set priority as:
1. `process.env.WDIO_LOG_LEVEL`
2. `options.logLevel`
3. `DEFAULT`

`DEFAULT` priority being higher than `process.env.WDIO_LOG_LEVEL` makes no sense

Reply:
That works for me 👍

Reply:
Fixed:

```ts
const envLogLevel = process.env.WDIO_LOG_LEVEL as Options.WebDriverLogTypes | undefined
options.logLevel = envLogLevel ?? options.logLevel // before validateConfig, so we could validate process.env.WDIO_LOG_LEVEL
const params = validateConfig(DEFAULTS, options)
```